docs/src/main/paradox/api-reference/operating-on-resources.md

```diff
@@ -137,9 +139,10 @@ GET /v0/{collection_address}
 - `{full_text_search_query}`: String - can be provided to select only the resources in the collection that have
 attribute values matching (containing) the provided token; when this field is provided the results will also include
 score values for each result
+- `{context}`: JsonLd - can be provided to define extra prefix mappings. By default, the search core context will be used. If this filed is provided, its content will be merged with the search core context.
```

Comment:
typo, `filed` instead of `field`. I think `parameter` is more correct here anyway.